themes/custom/az_barrio/css/style.css

```diff
@@ -6,6 +6,7 @@
  * of these @include files will be combined into a single file.
  */
 
+
```

Comment:
Do we care about the extra added space here? 😬

Reply:
```suggestion
```

Reply:
would be better to remove it, if possible.
```suggestion
```